iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -159,25 +160,20 @@ public static Collection inputs() throws Exception
     // How much to wait until a message makes it to the server, in milliseconds
     protected static final Integer SEND_TIMEOUT_MILLISECONDS = 60000;
 
-    public enum STATUS
```

Comment:
This status class was used to simplify twin callbacks, but it hid useful information for failure cases. Knowing why a callback fired back with an unsuccessful state is important to log when the test fails